lib/chef/resource/windows_package.rb

```diff
@@ -26,13 +26,12 @@ class Resource
     class WindowsPackage < Chef::Resource::Package
       include Chef::Mixin::Uris
 
-      provides :package, os: "windows"
       provides :windows_package, os: "windows"
+      provides :package, os: "windows"
```

Comment:
These lines are correct, imo.  `:windows_package` is only usable on windows, and its wired up as `:package` only on windows as well.   At some point it might be nice to unload the windows_package class on linux/solaris/aix/bsd since its just clutter there.

Note that people can do things like install apt-get on redhat, so pruning apt_package to platform_family of :debian is wrong and too strict (and some crazy person might have ported it to solaris even, so i was not overly strict when i set these things up).  The window stuff was deliberately restricted to windows though.